app/src/data/servey.ts

```diff
@@ -0,0 +1,99 @@
+export const mySurveys = [
+  {
+    id: 1,
+    title: '이색 콜라보 식품 구매 경험 및 인식',
+    respondent: 82,
+    recruitment: 100,
+    deadLine: '2021.12.12',
+    gift: 'etc',
+  },
+  {
+    id: 2,
+    title: '이색 콜라보 식품 구매 경험 및 인식',
+    respondent: 50,
+    recruitment: 100,
+    deadLine: '2021.12.12',
+    gift: 'coffee',
+  },
+  {
+    id: 3,
+    title: '이색 콜라보 식품 구매 경험 및 인식',
+    respondent: 50,
+    recruitment: 100,
+    deadLine: '2021.12.12',
+  },
+];
+
+export const interestSurveys = [
+  {
+    id: 1,
+    title: '제로(0kcal)음료에 대한 인식',
+    tag: '#20~30대  #일상',
```

Comment:
여러개의 값이니 tag 를 array 로 관리하는 것이 좋을 것 같아요! 
```suggestion
    tag: ['#20~30대',  '#일상'],
```